### ISSUE TITLE
Auto-email: Search function not working

### ISSUE THREAD

**Issue by masarapmabuhay** [usbong/UsbongStore#19]
Hi Zent,
I'm now using auto-email to send recommendations to our customers.

I noticed though that the search function to pick the product items that I'd want to recommend isn't working as it should.

When I type any of the following, auto-email doesn't show any search results: 
"Steve Jobs", "steve jobs", "%jobs%", etc.

What could be the probable cause?

Thank you!

Peace,
Mike

**Comment by zent-lim** [comment]
Hi Mike,

Please see attached files for my trial on your search queries.

Everything seemed to return ok except for %jobs% which is expected to
return nothing.

============
REQUEST
============

(1) Could you check if the entry is in the product table of the server
being used?

For my local environment, the entries "Steve Jobs"  and "PROMO-Steve Jobs,
The Innovators " have ids 7 and 35.

(2) Could you check if your search box has leading or trailing white
spaces? If yes that would cause an empty result because the white space is
also searched.

Thank you!

**Comment by masarapmabuhay** [comment]
Hi Zent,
Thank you for the reply.

1) Yes, the entry, "Steve Jobs" is in the product table of the prod server.
2) Yes, "Steve Jobs" and "PROMO-Steve Jobs" also have the IDs 7 and 35 respectively.
3) There are no leading or trailing white spaces.

I've verified the search box again.

It seems that it is now working after creating a new email.

I encountered the error after I was almost done picking my product entries.

Perhaps, it is only a rare occurrence.

Thank you for the advice.

Peace,
Mike
